Diagnostics/HealthChecker/DataCollection/ExchangeInformation/Get-FIPFSScanEngineVersionState.ps1

```diff
@@ -0,0 +1,147 @@
+﻿# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+. $PSScriptRoot\..\..\..\..\Shared\Invoke-ScriptBlockHandler.ps1
+
+Function Get-FIPFSScanEngineVersionState {
+    [CmdletBinding()]
+    [OutputType("System.Bool")]
+    param (
+        [Parameter(Mandatory = $true)]
+        [string]
+        $ComputerName,
+        [Parameter(Mandatory = $false)]
+        [scriptblock]
+        $CatchActionFunction
+    )
+
+    begin {
+        Write-Verbose "Calling: $($MyInvocation.MyCommand)"
+        function GetItemFromExchangeInstallPath {
+            param(
+                [string]$ExchangeSubDir
+            )
+
+            Write-Verbose "Calling: $($MyInvocation.MyCommand)"
+
+            $finalPathToQuery = Join-Path $env:ExchangeInstallPath $ExchangeSubDir
```

Comment:
Can't use `$env:ExchangeInstallPath` as that isn't set unless EMS is launched. Plus, it can be in a different location on remote machines.

Reply:
`ExchangeInstallPath` is a environment variable and so can be used without EMS: 
![image](https://user-images.githubusercontent.com/40993616/147957250-3a8f483d-76a2-4334-bdbe-a6db59812e57.png)

Reply:
Providing that if it is still there, we have seen cases where that isn't there. In the end it would be better to look for the path via the registry.

Reply:
Okay, we'll try to get the path via `Get-ItemProperty` and if it doesn't work we'll fallback to the `$env:ExchangeInstallPath`